### PR TITLE
Internal subnet

### PR DESCRIPTION
**Pull request by Op3rat0r**
Adding the internal subnet to the nc configuration. Helped to access the nc instance via my revers proxy. It was accesible from outside, just not from inside.

### CODE REVIEW

**Comment by Op3rat0r** [comment]
It's ok to just put the external domain into the trusted_proxy list.

**Comment by andryyy** [comment]
The trusted proxy is set by the helper script, isn't it?

**Comment by Op3rat0r** [comment]
In the array are two entries.
0 => 'fd4d:6169:6c63:6f77::/64',
1 => '172.22.1.0/24',

It works fine when I called the url from outside my network. But didn't work when you call the url from the internal nw. After that changes it works.

**Comment by andryyy** [comment]
Ah, okay, misunderstood it. :)

File: docs/third_party-nextcloud.md [hunk 1]
```diff
@@ -75,3 +75,31 @@ If you have previously used Nextcloud without mailcow authentication, but with t
...
+
+---
+
+## Update
+
+The Nextcloud instance can be updated easily with the web update mechanism. In the case of larger updates, there may be further changes to be made after the update. After the Nextcloud instance has been checked, problems are shown. This can be e.g. missing indices in the DB or similar.
+It shows which commands have to be executed, these have to be placed in the php-fpm-mailcow container.
+
+As an an example run the following command to add the missing indices.
+`docker exec -it -u www-data $ (docker ps -f name = php-fpm-mailcow -q) bash -c "php / web / nextcloud / occ db: add-missing-indices"`
```
Comment (andryyy) [inline docs/third_party-nextcloud.md:87]: Are there spaces between the path? Like "php / web / nextcloud"?

Comment (Op3rat0r) [reply docs/third_party-nextcloud.md:87]: Oh, no there are no spaces. It's a pathe "/web/nextcloud/occ"

docker exec -it -u www-data $(docker ps -f name=php-fpm-mailcow -q) bash -c "php /web/nextcloud/occ db:add-missing-indices"

**Comment by andryyy** [comment]
I removed the spaces. :) Thank you!

**Comment by Op3rat0r** [comment]
You are welcome! I am happy that I could contribute something to the cow ;)

**Pull request merged**